index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>The Quest</title>
+<script src="script.js" type="text/navascript"></script>
```

Comment:
Your script was not running at first, then I noticed an spelling error. It should be `type="text/javascript"`. Not sure how that happened :)